akka-cluster/src/multi-jvm/scala/akka/cluster/SurviveNetworkInstabilitySpec.scala

```diff
@@ -36,6 +36,7 @@ object SurviveNetworkInstabilityMultiJvmSpec extends MultiNodeConfig {
   commonConfig(debugConfig(on = false).withFallback(
     ConfigFactory.parseString("""
       akka.remote.system-message-buffer-size=100
+      akka.remote.artery.advanced.system-message-buffer-size=100
       akka.remote.netty.tcp.connection-timeout = 10s
       """)).
     withFallback(MultiNodeClusterSpec.clusterConfig))
```

Comment:
don't you need the special assert in this test also, as you did in SplitSpec?
wasn't the issue in this test originally?